query_server/query/src/execution/ddl/show_database.rs

```diff
@@ -0,0 +1,28 @@
+use crate::execution::ddl::DDLDefinitionTask;
+use async_trait::async_trait;
+use snafu::ResultExt;
+use spi::catalog::MetaDataRef;
+use spi::query::execution;
+use spi::query::execution::{ExecutionError, Output, QueryStateMachineRef};
+
+pub struct ShowDatabaseTask {}
```

Comment:
`ShowDatabases` instead of `ShowDatabase`

Reply:
nice hint